src/components/modes/student/StudentButtons.js

```diff
@@ -32,19 +41,28 @@ class StudentButtons extends Component {
     dispatchPostAppInstanceResource: PropTypes.func.isRequired,
     dispatchPatchAppInstanceResource: PropTypes.func.isRequired,
     dispatchRunCode: PropTypes.func.isRequired,
+    dispatchOpenInputSettings: PropTypes.func.isRequired,
+    dispatchCloseInputSettings: PropTypes.func.isRequired,
     currentCode: PropTypes.string.isRequired,
+    currentInput: PropTypes.string.isRequired,
+    programmingLanguage: PropTypes.string.isRequired,
     savedCode: PropTypes.string,
+    savedInput: PropTypes.string,
     feedback: PropTypes.string,
     inputResourceId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    stdinResourceId: PropTypes.string,
     userId: PropTypes.string,
     view: PropTypes.string,
+    isInputDisplayed: PropTypes.bool.isRequired,
```

Comment:
Usually we do not prepend `is` to booleans, so it should be `inputDisplayed`

Reply:
DONE.